references-in-rfcxml.md

```diff
@@ -9,80 +9,233 @@ dateCreated: 2021-11-04T23:45:18.949Z
 ---
 
 # Introduction
-RFCXML uses references that are encoded in a format called BibXML, as defined in RFC 7991. To simplify the process of inserting references, the IETF maintains a set of online citation libraries as follows:
+RFCXML uses references that are encoded in a format called BibXML, as defined in [RFC 7991](https://www.rfc-editor.org/info/rfc7991). To simplify the process of inserting references, the IETF maintains a set of online citation libraries as follows:
```

Comment:
Our wiki automagically adds links to RFC, STD and BCP references, so these should not be added manually.